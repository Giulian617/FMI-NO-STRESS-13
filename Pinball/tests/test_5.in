9 1
2 5 6
6 5 7
8 6 5
4 2 10
3 5 2
1 4 8
9 6 6
7 4 9
