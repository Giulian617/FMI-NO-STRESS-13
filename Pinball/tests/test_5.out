280456388
244540549
104987119
408661517
961354442
